docs/configuration_file.md

```diff
@@ -110,3 +110,47 @@ Some options are also mandatory.
 * `memory_limit` (*optional*, `string`) — memory limit to apply to build (for more info, see [documentation for resources](https://github.com/projectatomic/osbs-client/blob/master/docs/resource.md)
 
 * `storage_limit` (*optional*, `string`) — storage limit to apply to build (for more info, see [documentation for resources](https://github.com/projectatomic/osbs-client/blob/master/docs/resource.md)
+
+
+## Build JSON Templates
+
+In the `build_json_dir` there must be `prod.json` and `prod_inner.json` which
+defines the [OpenShift Build](https://docs.openshift.org/latest/dev_guide/builds.html)
+specification that will be used to enable the specific
+[atomic-reactor](https://github.com/projectatomic/atomic-reactor) plugins that
+should be enabled and the config values for each.
+
+There is also a third file that is optional that can exist along side the
+previous two in `build_json_dir` which is `prod_customize.json` and it will
+provide the ability to set site-specific customizations such as removing,
+plugins, adding plugins, or modifying arguments passed to existing plugins.
+
+The syntax of `prod_customize.json` is as follows:
+
+```json
+{
+    "disable_plugins": [
+        ["plugin_type1", "plugin_name1"],
+        ["plugin_type2", "plugin_name2"]
+    ],
```

Comment:
Would require some additional handling in code, but wouldn't be better to make this consistent with `enable_plugins`, and have
```json
"disable_plugins": [
    {
        "plugin_type1": "foo_type",
        "plugin_name1": "foo_name"
    },
    {
        "plugin_type2": "bar_type",
        "plugin_name2": "bar_name"
    }
]
```